SimGeneral/TrackingAnalysis/python/trackingParticleNumberOfLayersProducer_cff.py

```diff
@@ -0,0 +1,3 @@
+from SimGeneral.TrackingAnalysis.trackingParticleNumberOfLayersProducer_cfi import *
+from Configuration.StandardSequences.Eras import eras
+eras.fastSim.toModify(trackingParticleNumberOfLayersProducer, simHits=['famosSimHits:TrackerHits'])
```

Comment:
For the following I assume a goal of removing the introduced `trackingParticleNumberOfLayersProducer_cff.py` (that's my only problem with the approach above). With two different modules, it would be then up to the client(s) to do the customization for FastSim, either applying `toReplaceWith()` to the module itself, or modifying sequences+InputTags (I'm testing the latter approach because I feel that applying `toReplaceWith()` to an object elsewhere where the object is defined is a bit dirty). As long as there is one client (`TrackValidation_cff`) I think this is fine, but if there were at least two, this would lead to duplication.

Duplication could be removed in the `_cff.py` by applying `toReplaceWith()` to the module there. Then the only improvement I see would be that defaults for both FullSim and FastSim would be in a single file. Is this then what I should do?